src/components/LoanComponents/APR.tsx

```diff
@@ -0,0 +1,21 @@
+import React,{useContext, useEffect, useState} from "react";
+import { AppContext } from "../../ValueProvider";
+
+
+export default function APRfield() {
+    const {setAprValue} = useContext(AppContext)
+    const [APR,setAPR]= useState(0)

```

Comment:
Instead of state useContext values and dispatchers.

Reply:
changed.